buddyboss/buddyboss-platform
# Mobile App Integration Button and video added

### All Submissions:







### How to test the changes in this Pull Request:

Simply check "View DEMO" button here:
https://domain.com/wp-admin/admin.php?page=bp-integrations

Make sure this shows Michael Mobile App latest video.

### Proof Screenshots or Video

https://prnt.sc/v6kpuo

https://prnt.sc/v6kq8l

### Other information:
